src/routes/Auth.svelte

```diff
@@ -28,7 +28,7 @@
 <div class="flex h-dvh">
 	<div class="m-auto flex flex-col items-center gap-2">
 		<div class="inline-flex items-center gap-2">
-			<img src="logo.svg" alt="Lyntr" class="pointer-events-none h-40 w-40 select-none" />
+			<img src="/logo.svg" alt="Lyntr" class="pointer-events-none h-40 w-40 select-none" />
```

Comment:
I don't think that makes a difference.

Reply:
I had an issue where the logo wouldn't load on pages other than /